### PR TITLE
[Localization] Add pipeline template

### PR DESCRIPTION
**Pull request by vmapetr**
This PR adds a pipeline template and related scripts for the translation flow.

### CODE REVIEW

File: Localize/bump-versions.js [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+        .filter(p => fs.existsSync(path.resolve(p)) && fs.statSync(path.resolve(p)).isDirectory())
+        .forEach(p => pathSet.add(p));
+
+    for (let excludedTask of exclusions) {
+        pathSet.delete(`${rootFolder}${excludedTask}`);
+    }
+    return Array.from(pathSet);
+}
+
+function getTasksPaths(paths) {
```
Comment (max-zaytsev) [inline Localize/bump-versions.js:25]: We don't have Tasks folder in this repo

**Review by LiliaSabitova — APPROVED** [review]

File: Localize/bump-versions.js [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+    }
+
+    const fileList = getChangedFilesList();
+
+    const tasksPaths = getTasksPaths(fileList)
+    const commonPackages = getCommonPacks(fileList)
+
+    tasksPaths.forEach(taskPath => {
+        bumpTaskVersion(taskPath, currentSprint);
+    });
```
Comment (KonstantinTyukalov) [inline Localize/bump-versions.js:107]:
```suggestion
    const commonPackages = getCommonPacks(fileList)
```

File: Localize/bump-versions.js [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+}
+
+function getCommonPacks(paths) {
+    const packFolder = 'common-npm-packages/';
+    const exclusions = ['_download', 'node_modules', 'build-scripts'];
+
+    return getTaskOrPackagePaths(paths, packFolder, exclusions);
+}
+
+function bumpTaskVersion(taskPath, minor) {
```
Comment (KonstantinTyukalov) [inline Localize/bump-versions.js:39]: I believe we don't need this anymore

File: Localize/bump-versions.js (hunk 1 above)
Comment (vmapetr) [reply Localize/bump-versions.js:25]: Yep - the idea here is to reuse the same scripts that we use in another repos for now, and rework it later as an internal common package that includes scripts for version bump, sending messages, etc.

File: Localize/bump-versions.js (hunk 3 above)
Comment (vmapetr) [reply Localize/bump-versions.js:39]: We can do this, but please consider this
https://github.com/microsoft/azure-pipelines-tasks-common-packages/pull/51#discussion_r1046346042

Comment (KonstantinTyukalov) [reply Localize/bump-versions.js:39]: ok, sounds good

**Review by KonstantinTyukalov — APPROVED** [review]

**Review by max-zaytsev — APPROVED** [review]

**Review by kirill-ivlev — APPROVED** [review]

**Pull request merged**